Teapot/Teapot.swift

```diff
@@ -2,12 +2,7 @@ import Foundation
 
 /// A light-weight abstraction for URLSession.
 open class Teapot {
-
-    public enum TeapotError: Error {
-        case invalidRequestPath
-        case invalidResponseStatus
-        case missingImage
-    }
+    open static var localizationBundle = Bundle.init(for: Teapot.self)
```

Comment:
init ? ..

Reply:
Nice catch. Swift autocomplete is kinda shitty so I usually do `init(…autocomplete` and remove the `init` part. 👁